packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -323,7 +323,7 @@ async function fetchLatestSnapshotCore(
                     encodedBlobsSize,
                     sequenceNumber,
                     ops: snapshot.ops?.length ?? 0,
-                    nonSysOps: snapshot.ops?.filter((op) => !isSystemMessage(op)).length ?? 0,
+                    nonSysOps: snapshot.ops?.filter((op) => isClientMessage(op)).length ?? 0,
```

Comment:
I'd suggest using here IsRuntimeMessage and changing name of property from nonSysOps to userOps.
It's makes it non-mechanical change, but provides better value for this telemetry, as we really want how many user ops are there in files.

Reply:
And actually, it's more correct to use IsRuntimeMessage here, as it actually becomes mechanical change (i.e. equivalent to what it meant before change).
I think isClientMessage() should be only used in DeltaManager where we check for message to have clientId on it, as all client message have to have it. Otherwise isClientMessage() should  not be used in code base, I think, as I do not see good usage pattern for it.
Maybe add an assert that op being sent is isClientMessage() in ConnectionManager.sendMessages - that would be useful to make it symmetrical with check in DeltaManager - we check on sending and receiving ops for being client ops.